core/src/main/java/de/danoeh/antennapod/core/feed/Feed.java

```diff
@@ -392,7 +389,7 @@ public void setLink(String link) {
     }
 
     public String getDescription() {
-        return description;
+        return de.danoeh.antennapod.core.util.TextUtils.getNormalizedText(description);
```

Comment:
Parsing html should be done outside of `Feed.java`. `Feed` is the model that represents the actual feed. How we display the feed afterwards does not belong to the feed itself, in my opinion.